### ISSUE TITLE
too big report size

### ISSUE THREAD

**Issue by rayenmessaoudi** [cuckoosandbox/cuckoo#2211]
##### My issue is:
when i m trying to scan a blank pdf file the report.json file is too big (338  MO) can't be stored in the database :(
##### My Cuckoo version and operating system are:
version : 2.0.5
OS : ubuntu 16.04

##### The log, error, files etc can be found at:

the report.json can be found here
https://drive.google.com/file/d/1tbkRqRF92X5Ar1gWOmE8lhjXKtS65PJC/view?usp=sharing

**Comment by rayenmessaoudi** [comment]
yes in order to enrich my cuckoo analysis and for better score calculation

**Comment by doomedraven** [comment]
sorry i have bad news for you, we aware of this problem, so you need to remove yara which you don't need as mongo has his own limits which is 16mb, and can't be changed

**Comment by rayenmessaoudi** [comment]
but the document i scanned is not malicious why yara act in this case

**Comment by doomedraven** [comment]
poor yaras

**Comment by rayenmessaoudi** [comment]
i m using the famous yara rules repository in github

**Comment by doomedraven** [comment]
if you do search in issues you will see what all uses it :D and have the same error, `famous yara rules repository in github` != optimized/perfect

**Comment by rayenmessaoudi** [comment]
can you explain plz why this repo is not perfet and sorry for the disturbance
